src/chain_spec.rs

```diff
@@ -1174,3 +1174,137 @@ pub mod mainnet {
         )
     }
 }
+
+pub mod ci {
+    use super::*;
+    use polymesh_runtime_ci::{self as rt, constants::time};
+
+    pub type ChainSpec = sc_service::GenericChainSpec<rt::runtime::GenesisConfig>;
+
+    session_keys!();
+
+    fn genesis(
+        initial_authorities: Vec<InitialAuth>,
+        root_key: AccountId,
+        _enable_println: bool,
+        treasury_bridge_lock: BridgeLockId,
+        key_bridge_locks: Vec<BridgeLockId>,
+    ) -> rt::runtime::GenesisConfig {
+        let (identities, stakers, complete_txs) = genesis_processed_data(
```

Comment:
You may want to use `dev_genesis_processed_data` instead if you want to be closer to our usual `dev` chains. The difference is in the initial set of identities etc. which are set up on genesis.

Reply:
nope, it should be the same as `testnet-dev` which is what the current integration tests are based on and `testnet-dev` uses genesis_processed_data